### DISCUSSION TITLE
两个问题

### DISCUSSION THREAD

**Discussion by RudeTheJack** [hooke007/mpv_PlayKit#139] [Q&A 问答] [answered]
我shift+回车可以看到文件夹里面的其他视频，但是我的这个按键一直都是灰色的不能用
![屏幕截图 2022-06-09 231238](https://user-images.githubusercontent.com/27954602/172882032-7a49d757-76b0-41d1-beb7-bf60ccb5c084.png)

还有一个问题，mpv不能播放.ts和.iso文件吗

希望大佬能帮我解答一下，谢谢！

**Comment by hooke007** [comment, marked as answer]
标题被我改掉了，没看出和我的版本有什么特殊联系。。。
这是章节按钮。
能。

**Reply by RudeTheJack** [reply]
我这边播放不了.ts文件，然后.iso文件的支持也非常差，几乎等于播不了，是我电脑硬件问题吗？然后这个章节按钮要怎么设置才能亮起来啊，谢谢大佬🙏

**Reply by hong-tm** [reply]
> 我这边播放不了.ts文件，然后.iso文件的支持也非常差，几乎等于播不了，是我电脑硬件问题吗？然后这个章节按钮要怎么设置才能亮起来啊，谢谢大佬🙏

要在视频里面设置了章节这东西才能用，没有章节自然不会亮。

**Reply by RudeTheJack** [reply]
那能不能让这个按键变为播放同一文件夹里的下一个视频的功能啊

那iso和ts文件播放不了是我显卡的问题吗，GTX650😥

**Reply by hong-tm** [reply]
要看视频有没有章节， mpc-hc 和 mpv都可以， 不是每个视频都有章节的

> 那能不能让这个按键变为播放同一文件夹里的下一个视频的功能啊

在input.conf 里面改

```
 <                    add chapter -1               # 章节 -
 >                    add chapter  1               # 章节 +
 CTRL+,               playlist-prev                # 列表中的上个文件
 CTRL+.               playlist-next                # 列表中的下个文件
```

去这个

```
 <                      playlist-prev  
 >                    playlist-next         
 CTRL+,               add chapter -1                
 CTRL+.               add chapter  1               
```

> 那iso和ts文件播放不了是我显卡的问题吗，GTX650😥

我之前播放没有问题就是ts文件会缺帧

**Reply by hong-tm** [reply]
> 这个章节按钮要怎么设置才能亮起来啊，谢谢大佬🙏


章节一般上 bdrip会有人做，webrip可能没有，进度条会显示的
![image](https://user-images.githubusercontent.com/51999870/173027752-e6f257e7-ced7-4c3a-8239-594ba297382c.png)

**Reply by RudeTheJack** [reply]
哇，太细致了，谢谢大佬🙏我有一个ts文件，potplayer能播，mpv是根本打不开😂，iso文件看情况，有的mpv能播，有的不行，而且就算能播的也不会显示进度条缩略图，所以我iso都是用的其他播放器

**Reply by hong-tm** [reply]
> 哇，太细致了，谢谢大佬🙏我有一个ts文件，potplayer能播，mpv是根本打不开😂，iso文件看情况，有的mpv能播，有的不行，而且就算能播的也不会显示进度条缩略图，所以我iso都是用的其他播放器

### 我这边没问题
### 我的显卡也一般😭
![image](https://user-images.githubusercontent.com/51999870/173029295-5e745a08-54bc-481b-8fcf-a9d2ca50d4c5.png)

**Reply by hong-tm** [reply]
> 哇，太细致了，谢谢大佬🙏我有一个ts文件，potplayer能播，mpv是根本打不开😂，iso文件看情况，有的mpv能播，有的不行，而且就算能播的也不会显示进度条缩略图，所以我iso都是用的其他播放器

## 文件发上来看看，链接

**Reply by RudeTheJack** [reply]
难道是因为我的这个ts文件有问题？大佬你这个ts文件能不能传我一份我试试😂

**Reply by hong-tm** [reply]
> 哇，太细致了，谢谢大佬🙏我有一个ts文件，potplayer能播，mpv是根本打不开😂，iso文件看情况，有的mpv能播，有的不行，而且就算能播的也不会显示进度条缩略图，所以我iso都是用的其他播放器

可以尝试后缀改去mp4,有时候可以解决问题。

**Reply by RudeTheJack** [reply]
> 要看视频有没有章节， mpc-hc 和 mpv都可以， 不是每个视频都有章节的
> 
> > 那能不能让这个按键变为播放同一文件夹里的下一个视频的功能啊
> 
> 在input.conf 里面改
> 
> ```
>  <                    add chapter -1               # 章节 -
>  >                    add chapter  1               # 章节 +
>  CTRL+,               playlist-prev                # 列表中的上个文件
>  CTRL+.               playlist-next                # 列表中的下个文件
> ```
> 
> 去这个
> 
> ```
>  <                      playlist-prev  
>  >                    playlist-next         
>  CTRL+,               add chapter -1                
>  CTRL+.               add chapter  1               
> ```
> 
> > 那iso和ts文件播放不了是我显卡的问题吗，GTX650😥
> 
> 我之前播放没有问题就是ts文件会缺帧

我按照这样设置了之后我的那个按键还是灰色的😭

**Reply by hong-tm** [reply]
> 

这个是给快捷键用的， 按键的我也不熟，要问上面的大佬

**Reply by hong-tm** [reply]
> 难道是因为我的这个ts文件有问题？大佬你这个ts文件能不能传我一份我试试😂

https://anonfiles.com/F4838aoby9/_1_-_-_AGE_ts

### 可以尝试后缀改去mp4,有时候可以解决问题。

**Reply by RudeTheJack** [reply]
> > 
> 
> 这个是给快捷键用的， 按键的我也不熟，要问上面的大佬

😭好吧，谢谢你花时间告诉我这么多👍

**Reply by hooke007** [reply]
最外部的两按钮就是切换文件用的，为什么要去改章节按钮
ts需要封装成mp4确保跳转功能完全可用
iso我没有条件去大量测试

**Reply by RudeTheJack** [reply]
是的，我刚刚才发现了这俩按键😂关于视频格式我已经不纠结了，谢谢大佬的解答👍
